src/modules/newList.js

```diff
@@ -1,12 +1,21 @@
-/* eslint-disable linebreak-style */
+/* eslint-disable no-plusplus */
```

Comment:
- Hey, as this is considered a bad practice, let's try to write code that conforms to the linter. please, kindly remove this eslint disable and try fixing the error.